tests/test_clnrest.py

```diff
@@ -273,6 +273,58 @@ def message(data):
     assert len([n for n in notifications if n.find('invoice_creation') > 0]) == 1
 
 
+def test_clnrest_websocket_rune_readonly(node_factory):
```

Comment:
oh nice, I think we should fix https://github.com/ElementsProject/lightning/issues/6725 also in this release

Reply:
This would be great :)